docs/source/api.rst

```diff
@@ -0,0 +1,6 @@
+API
+===
+
+.. autofunction:: ome_zarr.writer.write_image
```

Comment:
Potentially for a follow-up https://github.com/ome/omero-rois/blob/master/docs/index.rst uses `automodule` which would have the benefit of avoiding having to include every API by hand